### PR TITLE
Improvement: egg hunt include note

### PR DESCRIPTION
**Pull request by hannibal002**
## What
Sharing the name of the rabbit hunt egg note in chat.

## Changelog Improvements
+ Sharing the name of the rabbit hunt egg note in chat. - hannibal2

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt [hunk 1]
```diff
@@ -27,9 +27,14 @@ object HoppityEggsManager {
...
 
+    /**
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§don a ledge next to the stairs up§r§d!
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§aChocolate Dinner Egg §r§dbehind Emissary Sisko§r§d!
+     * REGEX-TEST: §d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§dnear the Diamond Essence Shop§r§d!
+     */
     private val eggFoundPattern by ChocolateFactoryAPI.patternGroup.pattern(
         "egg.found",
-        "§d§lHOPPITY'S HUNT §r§dYou found a §r§.Chocolate (?<meal>\\w+) Egg.*"
+        "§d§lHOPPITY'S HUNT §r§dYou found a §r§.Chocolate (?<meal>\\w+) Egg §r§d(?<note>.*)§r§d!"
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt:37]: will not match this:
§d§lHOPPITY'S HUNT §r§dYou found a §r§9Chocolate Lunch Egg §r§dnear the §r§5Dark Auction§r§d!
or
§d§lHOPPITY'S HUNT §r§dYou found a §r§6Chocolate Breakfast Egg §r§din §r§cUnincorporated Territory§r§d!

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
havent tested in game but the rest of the changes look good

File: src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt (hunk 1 above)
Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt:37]: i dont understand

Comment (CalMWolfs) [reply src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt:37]: > i dont understand

these two messages dont match your new pattern

Comment (CalMWolfs) [reply src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsManager.kt:37]: wait never mind i'm going crazy

**Review by CalMWolfs — APPROVED** [review]
nvm

File: src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsShared.kt [hunk 2]
```diff
@@ -32,16 +34,18 @@ object HoppityEggsShared {
             val eggLocation = LorenzVec(x, y, z)
 
             val meal = getEggType(event)
+            val note = groupOrNull("note")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsShared.kt]:
```suggestion
            val note = groupOrNull("note")?.removeColor()
```

**Review by CalMWolfs — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsShared.kt (hunk 2 above)
Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsShared.kt]: fair

**Review by CalMWolfs — APPROVED** [review]

**Pull request merged**